backend/test/modules/doubles-tournament/core/application/CreateTournamentWithTeams.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { testCreateTournamentWithTeamsModule } from './TestCreateTournamentWithTeamsModule';
+import { CreateTournamentWithTeams } from '../../../../../src/modules/doubles-tournament/core/application/command/CreateTournamentWithTeams';
+import { TournamentWithTeamsWasCreated } from '../../../../../src/modules/doubles-tournament/core/domain/event/TournamentWithTeamsWasCreated';
+import { TournamentTeam } from '../../../../../src/modules/doubles-tournament/core/domain/TournamentTeam';
+import { TeamIdGeneratorStub } from '../../../../test-support/shared/core/TeamIdGeneratorStub';
+
+describe('Create Tournament With Teams', () => {
+  it('given only 1 pair of players, when create tournament, then tournament was created with 1 team', async () => {
+    //Given
+    const currentTime = new Date();
+    const entityIdGen = TeamIdGeneratorStub(['TeamId']);
+    const tournamentCreation = testCreateTournamentWithTeamsModule(currentTime, entityIdGen);
```

Comment:
To chyba warto nazwać po prostu `doublesTournament` lub `doublesTournamentModule`, bo to oznacza właśnie cały moduł, który wykonuje komendy i publikuje eventy :)